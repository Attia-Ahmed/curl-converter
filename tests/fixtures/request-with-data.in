curl -H 'Content-Type: application/x-www-form-urlencoded' -d 'foo=bar&baz=qui' https://example.com
