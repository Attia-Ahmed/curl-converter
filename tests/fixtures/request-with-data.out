Http::asForm()
    ->post('https://example.com', [
        'foo' => 'bar',
        'baz' => 'qui',
    ]);
